clang/test/CIR/emit-actions.cpp

```diff
@@ -11,7 +11,7 @@
 
 int x = 1;
 
-// BC: @x = dso_local global i32 1
+// BC: @x = global i32 1, align 4
```

Comment:
I didn't get any buildbot notifications about this test failing. Where are you seeing this failure?

Reply:
Oh, I guess I wouldn't see buildbot failures, since we don't have a buildbot with ClangIR enabled. I haven't seen it fail locally though.

Reply:
This is happening when I run the tests inside Google's infra. I've updated the commit so that it works both inside and out.